bin/ls.py

```diff
@@ -77,96 +73,132 @@ def is_image(path):
         return True
     else:
         return False
-
+        
+def human_legible_size(num):
```

Comment:
I believe that `libcore.sizeof_fmt()` does the same, so you could instead use `_stash.libcore.sizeof_fmt()`.

Reply:
Yes it does. But it uses A LOT of characters to display the size and if you have ever used Pythonista on a mobile phone instead of an iPad, you know how precious the place on one line is.